recipe/meta.yaml

```diff
@@ -58,9 +62,7 @@ requirements:
     - cytoolz >=0.7.3  # [python_impl == 'cpython']
     # cloudpickle is necessary to provide the 'processes' scheduler for dask
     - cloudpickle >=0.2.1
-    # Using selector until packages for other platforms are (re)built using
-    # newer "defaults" toolchains that use the `_openmp_mutex` mechanism.
-    - _openmp_mutex  # [linux and (not ppc64le)]
+    - _openmp_mutex  # [linux]
```

Comment:
`_openmp_mutex` this seems to be a private variable. Normally we will avoid having private variables mentioned in the recipe.

Reply:
It's ok to use `_openmp_mutex` if it's required for some platforms

Reply:
This is a package, not variable, name. It's fine to use it here.